bundles/org.openhab.binding.onecta/src/main/resources/OH-INF/i18n/onecta.properties

```diff
@@ -0,0 +1,1234 @@
+thing-type.config.onecta.domestic-hotwatertank.unitID.label = Unit ID
+thing-type.config.onecta.domestic-hotwatertank.unitID.description = Diakin unit
+thing-type.config.onecta.gateway.unitID.label = Unit ID
+thing-type.config.onecta.gateway.unitID.description = Diakin unit
+thing-type.config.onecta.indoor-unit.unitID.label = Unit ID
+thing-type.config.onecta.indoor-unit.unitID.description = Diakin Indoor unit
+
+# channel group types
+
+channel-group-type.onecta.basic-group-gateway.label = Basic
+channel-group-type.onecta.basic-group-gateway.description = BasicGroup related information
+channel-group-type.onecta.basic-group-indoor-unit.label = Basic
+channel-group-type.onecta.basic-group-indoor-unit.description = BasicGroup related information
+channel-group-type.onecta.basic-group-watertank.label = Basic
+channel-group-type.onecta.basic-group-watertank.description = BasicGroup related information
+channel-group-type.onecta.basic-group.label = Basic
+channel-group-type.onecta.basic-group.description = Basic group related information
+channel-group-type.onecta.consumption-data-cooling-group.label = Consumption data cooling
+channel-group-type.onecta.consumption-data-cooling-group.description = Consumption data cooling
+channel-group-type.onecta.consumption-data-heating-group.label = Consumption data heating
+channel-group-type.onecta.consumption-data-heating-group.description = Consumption data heating
+channel-group-type.onecta.demandcontrol-group.label = Demandcontrol
+channel-group-type.onecta.demandcontrol-group.description = Demandcontrol related information
+channel-group-type.onecta.extra-group.label = Extra
+channel-group-type.onecta.extra-group.description = Extra special information
+
+# channel types
+
+channel-type.onecta.acunit-demand-control-fixed-max-value.label = Demand Control fixed Max value
+channel-type.onecta.acunit-demand-control-fixed-max-value.description = Max value of fixed demand control (readonly)
+channel-type.onecta.acunit-demand-control-fixed-min-value.label = Demand Control fixed Min value
+channel-type.onecta.acunit-demand-control-fixed-min-value.description = Min value of fixed demand control (readonly)
+channel-type.onecta.acunit-demand-control-fixed-step-value.label = Demand Control fixed Step value
+channel-type.onecta.acunit-demand-control-fixed-step-value.description = Step value of fixed demand control (readonly)
+channel-type.onecta.acunit-demand-control.label = Demand control
+channel-type.onecta.acunit-demand-control.description = Demand control
+channel-type.onecta.acunit-demand-control.state.option.OFF = Off
+channel-type.onecta.acunit-demand-control.state.option.AUTO = Auto
+channel-type.onecta.acunit-demand-control.state.option.FIXED = Fixed
+channel-type.onecta.acunit-demand-control.state.option.SCHEDULED = Scheduled
+channel-type.onecta.acunit-demand-controlfixedvalue.label = Demand Control fixed Value
```

Comment:
1. Why is `fixed` not capitalized, but the other words are?
2. Not sure if this is a fixed term in the context of this binding, but `Demand fixed control value` would sound more natural to me

Applies to other entries in this file as well.